styles/header.css

```
@@ -1,12 +1,91 @@
-.div--maincolor{
+.container__header {
+  padding: 20px 175px;
+  font-family: var(--font-hiragino);
+}
+
+.header {
   background-color: var(--bg-main-blue);
 }
 
-.h1 {
+
+.header-content {
+  display: grid;
+  width: 100%;
+  grid-template-columns: repeat(14, 1fr);
+  align-items: center;
+}
+
+.header-content__title {
+  grid-column: 1 / 6;
+  font-size: 12px;
+  line-height: 18px;
```

`line-height`の推奨`unit`は`unitがない`値ですね。
`line-height = pxのline-height/font-size`なので、今回は、`18/12=1.5`ですね。
```suggestion
  line-height: 1.5;
```

---

しかし、`line-height`が`1.6333333`のようは、分割できない数字の場合、pxのline-heightでも良いと個人的にしています。

---

[こちらのcommit](https://github.com/ShunDeveloper/prtimes_htmlcss/pull/10/commits/6a9dea799028b70ecbcf8a97dc86892820ebbc0e)で修正をいたしました.